esp/templates/program/modules/classsearchmodule/class_search.html

```diff
@@ -13,6 +13,9 @@
     <script type="text/jsx" src="/media/scripts/query-builder.jsx"></script>
     <script type="text/javascript" src="/media/scripts/program/modules/flag-results-page.js"></script>
     <script type="text/javascript" src="/media/scripts/program/modules/flag-edit.js"></script>
+    <!--jQuery.initialize plugin is created to help maintain dynamically created elements on the page-->
+    <script src="https://rawgit.com/pie6k/jquery.initialize/eb28c24e2eef256344777b45a455173cba36e850/jquery.initialize.js"></script>
```

Comment:
rawgit is going away soon (see rawgit.com, also #2630 ) so please don't add any new rawgit links. I don't know another solution other than to copy the js file directly into our codebase. (@benjaminjkraft ?)